src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
```suggestion
import { Person } from './components/Person';
```
Because you already created index.js inside the Person folder.